base/math.jl

```diff
@@ -1010,6 +1012,8 @@ For any other non-subnormal floating-point number `x`, this corresponds to the e
 
 See also [`signbit`](@ref), [`significand`](@ref), [`frexp`](@ref), [`issubnormal`](@ref), [`log2`](@ref).
 
+See also [`significand`](@ref), [`exponent`](@ref), [`ldexp`](@ref).
+
```

Comment:
These should be merged

```suggestion
See also [`signbit`](@ref), [`significand`](@ref), [`frexp`](@ref), [`issubnormal`](@ref), [`log2`](@ref), [`ldexp`](@ref).
```

Reply:
Merge conflict of sorts with #47085